From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Apr 2016 11:37:09 +0200
Subject: fix_build

===================================================================
---
 CMakeLists.txt          | 1 +
 umbrello/CMakeLists.txt | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 187f5dd..0b1341f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,6 +141,7 @@ else()
         KIO
         TextEditor
         WidgetsAddons
+        WindowSystem
         XmlGui
     )
 
diff --git a/umbrello/CMakeLists.txt b/umbrello/CMakeLists.txt
index dd5806d..daee0fe 100644
--- a/umbrello/CMakeLists.txt
+++ b/umbrello/CMakeLists.txt
@@ -545,7 +545,7 @@ else()
         ${CMAKE_CURRENT_SOURCE_DIR}/pics/global/hi128-apps-umbrello.png
     )
     add_executable(umbrello ${umbrello_SRCS})
-    target_link_libraries(umbrello libumbrello)
+    target_link_libraries(umbrello libumbrello KF5::WindowSystem)
 endif()
 if(LIB_BUILD_MODE EQUAL SHARED)
     install(TARGETS libumbrello ${INSTALL_TARGETS_DEFAULT_ARGS} )
